### PR TITLE
P1 -Feature - Rating component 

### PR DESCRIPTION
**Pull request by Koushith**
## Description

- [x] Adds Rating Component
## Screenshot and Code Snippets
image

### CODE REVIEW

File: src/components/rating/rating.component.html [hunk 1]
```diff
@@ -0,0 +1,163 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../../styles/navbar.css" />
+    <link rel="stylesheet" href="../../styles/sidebar.css" />
+    <link rel="stylesheet" href="../../../docs/getting-started.css" />
+    <link rel="stylesheet" href="../../../index.css" />
```
Comment (arulk8) [inline src/components/rating/rating.component.html:10]: CSS imports could be reduced

**Review by arulk8 — APPROVED** [review]

**Review by shubhmehra — APPROVED** [review]
format your code with prettier..
otherwise good to go

File: src/components/rating/rating.component.html (hunk 1 above)
Comment (Koushith) [reply src/components/rating/rating.component.html:10]: yes, will fix that in the final touches.

File: src/components/rating/rating.component.css [hunk 2]
```diff
@@ -0,0 +1,9 @@
+.star,
+i {
+  font-size: 2em;
+  color: var(--primary-main);
+}
```
Comment (anurag-kkumarr) [inline src/components/rating/rating.component.css:5]: Use one specific-units for font-size everywhere. Rem is good for responsiveness. em can give unexpected behviour sometimes.

**Review by anurag-kkumarr — COMMENTED** [review]
check suggested changes. Else It looks good

**Pull request merged**